src/NuGet.Core/NuGet.ProjectModel/LockFile/LockFileTarget.cs

```diff
@@ -61,5 +61,15 @@ public override int GetHashCode()
 
             return combiner.CombinedHash;
         }
+
+        public LockFileTarget Clone()
+        {
+            return new LockFileTarget
+            {
+                TargetFramework = TargetFramework,
+                RuntimeIdentifier = RuntimeIdentifier,
+                Libraries = new List<LockFileTargetLibrary>(Libraries)
```

Comment:
You need to clone the actual libraries too.

Reply:
will get rid of it